packages/titanium-dialog/src/titanium-dialog.ts

```diff
@@ -151,10 +153,8 @@ export class TitaniumDialogElement extends LitElement {
       box-shadow: 0 3px 5px -1px rgba(0, 0, 0, 0.2), 0 6px 10px 0 rgba(0, 0, 0, 0.14), 0 1px 18px 0 rgba(0, 0, 0, 0.12);
       -webkit-box-sizing: border-box;
       box-sizing: border-box;
-      -webkit-transform: scale(0.8);
```

Comment:
These styles create a new bounding box that essentially make fixed elements bound to this instead of the window.